common.css

```diff
@@ -4,12 +4,13 @@ body{
     background-color: var(--background-color);
     font-family: poppins;
     overflow-x: hidden;
-    max-width: 2000px;
+    max-width: 125rem;
     margin: auto;
 }
 :root{
     --light-blue: #f2fbff;
     --color-black: black;
```

Comment:
Better to use hex here as well for consistency.

Reply:
Done